core/services/chainlink/application.go

```diff
@@ -285,14 +291,29 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 	} else {
 		globalLogger.Debug("Off-chain reporting disabled")
 	}
+	terraORM := terra.NewORM(db, globalLogger, cfg)
 	if cfg.FeatureOffchainReporting2() {
 		globalLogger.Debug("Off-chain reporting v2 enabled")
+		// TODO: parameterize poll period
```

Comment:
If not addressed in this PR, lets have a link to a shortcut ticket

Reply:
I'm promoting `ConfirmMaxPolls` and `ConfirmPollPeriod` to the chain config in: https://github.com/smartcontractkit/chainlink/pull/5814